Dockerfile-tests

```diff
@@ -0,0 +1,12 @@
+FROM hseeberger/scala-sbt
```

Comment:
@zarsky-broad Is there a trusted source for these images? I didn't see an obvious one to trust (though I guess I'm equally skeptical of hseeberger and I don't know why we chose it here).